web/public/locales/it.json

```diff
@@ -163,9 +163,15 @@
     "production": "Produzione",
     "consumption": "Consumo",
     "cpinfo": "Il <b>consumo</b> considera importazioni ed esportazioni, la <b>produzione</b> li ignora",
+    "aggregateInfo": "Visualizza dati per zona o aggregati per nazione",
```

Comment:
I'm not a Italian speaker but does this match the same intention as:
"Toggle between a granular zone view and an aggregated country view"

It seems to miss the granular part?

Reply:
It's not the literal translation of the English one, but I think it would sound nicer to a native speaker.
Literally translated would be something like "View data grouped by zone or grouped by country"
I think it's already clear, but if you wanna stress the granular part more, it could be changed to
"Visualizza dettagli per zona o dati aggregati per nazione"

If you want something closer to the English version, another option could be:
"Alterna tra una vista dettagliata per zona e una vista aggregata per nazione"

Reply:
I think this will be fine. Let's get it merged!